templates/core/cafe.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Sharespots</title>
```

Comment:
same here, please extend our new templates instead of defining it all again in each one https://github.com/PolyglotDevsLondon/sharespots/blob/develop/templates/core/index.html